homeassistant/components/network/network.py

```diff
@@ -0,0 +1,237 @@
+"""Network helper class for the network integration."""
+from __future__ import annotations
+
+from ipaddress import ip_address
+import logging
+from typing import Any, Iterable, cast
+
+import ifaddr
+from pyroute2 import IPRoute
+
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.storage import Store
+
+from .const import (
+    ATTR_CONFIGURED_ADAPTERS,
+    DEFAULT_CONFIGURED_ADAPTERS,
+    NETWORK_CONFIG_SCHEMA,
+    STORAGE_KEY,
+    STORAGE_VERSION,
+)
+from .models import Adapter, IPv4ConfiguredAddress, IPv6ConfiguredAddress
+
+MDNS_TARGET_IP = "224.0.0.251"
```

Comment:
Why is it not in `const.py`?

Reply:
No good reason, just haven't moved it yet.